deploy/helm/kubecf/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: ~
+appVersion: v0.0.0
 description: A Helm chart for KubeCF
 name: kubecf
-version: ~
+version: v0.0.0
```

Comment:
I thought we had placeholder values here to make sure we _did_ replace them before helm sees it?

Reply:
Helm 3 doesn't like the nil values.

Reply:
Yeah, but I thought the point was to ensure we processed the files before helm got to it?

Reply:
No, this file is not a template. `helm package` takes the versions and apply here.